mmyolo/datasets/transforms/mix_img_transforms.py

```diff
@@ -330,6 +335,42 @@ def mix_img_transform(self, results: dict) -> dict:
         results['gt_ignore_flags'] = mosaic_ignore_flags
         return results
 
+    @autocast_box_type()
+    def no_mix_img_transform(self, results: dict) -> dict:
```

Comment:
```suggestion
    def no_mix_img_transform(self, results: dict) -> dict:
```

Reply:
PR #85 has delete this , pls check 😁
https://github.com/open-mmlab/mmyolo/pull/85/commits/657438b8188edde70de503493f2a70555423ec72